### PR TITLE
fix: button on mobile

### PR DESCRIPTION
**Pull request by riderx**
fix:
- fix button reactions, add touch event
- fix button mic, video, share screen with z-index
- fix position login form

### CODE REVIEW

**Comment by Donorhan** [comment]
I'm not a fan of using `screen.width` and `screen.height`, it returns the screen size without taking into account browser ui, side by side window opening, etc. The fix for the ipad is mainly done by changing the meta and form-account to `fixed`, right? The other changes are great, thanks!

**Comment by riderx** [comment]
@Donorhan i was following this https://stackoverflow.com/questions/4629969/ios-return-bad-value-for-window-innerheight-width
```
TL;DR: Use document.documentElement.clientHeight/Width

This is not exactly an answer to the question, but it's helpful information if you're trying to do things based on the size of the document shown in the browser.

window.innerWidth (and height) can get wacky on mobile browsers, even when you use the proper viewport meta tags. On some browsers it can be very inconsistent.

screen.width works better for mobile browsers, but then you need to take into account orientation and whether or not the browser is mobile or desktop (if you need to be responsive for both), as screen.width gives much more variation on desktop compared to window.innerWidth.

In my experience, document.documentElement.clientWidth is the best way to go. Mobile browsers are much more consistent for this attribute than for window.innerWidth, and since you're dealing with the dimensions of the html element and not the browser window or the device screen, it's consistent between mobile and desktop, and orientation does not need to be accounted for.
```

The best solution is maybe `document.documentElement.clientWidth`

**Comment by riderx** [comment]
I did the switch to the new solution

**Comment by riderx** [comment]
Screenshot 2021-09-24 at 11 58 58
this is failing with `document.documentElement.clientWidth` but was working with `screen.width`.
`screen.width` was working for me in my mac browser ui, side by side window opening and ipad too .
I will revert.

**Comment by riderx** [comment]
@Donorhan i reverted most of the changes, and just keept the mobile fix for buttons

**Comment by Donorhan** [comment]
Thanks a lot!
